.github/workflows/build-and-test.yml

```diff
@@ -46,6 +48,18 @@ jobs:
                   override: true
             - name: Run ffi circom tests
               run: cd mopro-ffi && cargo test --features circom --no-default-features
+    test-ffi-nova_scotia:
```

Comment:
can we align the usage of `-` and `_`
I prefer the feature flag be `nova-scotia` 
e.g. `--features nova-scotia`
(but in rust code some place needs to be `nova_scotia` then just keep them `nova_scotia`)